src/tests/unit/helpers/find.test.ts

```diff
@@ -0,0 +1,177 @@
+import * as findHelpers from '../../../../src/helpers/find'
+import { Ability, AllSyncStorageLayer, SlotId } from '../../../types';
+
+interface MockUserState {
+    firstName: string,
+    lastName: SVGAnimatedString,
+    age: number
+}
+
+describe('findIndexofSlotIdsBySlotId function', () => {
+    test('when SlotID does not exist', () => {
+        const abilities: Ability<MockUserState, AllSyncStorageLayer<MockUserState>>[] = [{
```

Comment:
Remove `abilities`

Reply:
removed abilities and unnecessary comments